build/ci/templates/dotnet-install.yml

```diff
@@ -1,5 +1,17 @@
 steps:
 
+  - bash: |
+      rm global.json
+      cp build/ci/net7/global.json global.json
```

Comment:
shouldn't this `global.json` file be named differently than the .NET 8 global json file?

Reply:
Do you mean the one at repo root or in `net7` directory?

The one at repo root must be `global.json` for it to be recognized and used when building. The other ones in `net7` and `net8` directories can be named anything, at the end they are copied to the repo root with the name `global.json`. I kept the naming `global.json` for simplicity and they are differentiated based on the directory they are in.

We could have them as `build/ci/global-net7.json` and `build/ci/global-net8.json` if we wanted.

Reply:
Thanks for the explanation @Youssef1313 . My question is more related to the copy command. I see

```
cp build/ci/net7/global.json global.json
```
and 

```
cp build/ci/net8/global.json global.json
```

Isn't that later `cp` overriding the first one? since both files keep the same name on the destination "global.json"?

Reply:
@ajpinedam Only one of those commands should get executed. The conditions are `eq(variables.UnoDisableNetCurrent, 'true'))` and `ne(variables.UnoDisableNetCurrent, 'true'))`

Reply:
Thanks!